htdocs/web_portal/controllers/scopes.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /*______________________________________________________
  *======================================================
- * File: view_user.php
- * Author: George Ryall, David Meredith
- * Description: Retrieves and draws the data for a user
+ * File: view_service_types.php
+ * Author: George Ryall, David Meredith, Elliott Kasoar
+ * Description: Controller for showing all scope tags available in GOCDB
  *
  * License information
  *
- * Copyright 2013 STFC
+ * Copyright � 2013 STFC
```

Comment:
Ah interesting, it comes from [here](https://github.com/GOCDB/gocdb/blob/dev/htdocs/web_portal/controllers/admin/scopes.php#L10). I basically didn't touch the license info, but it's decided that the final file is closer to scope_help.php, so it looks like I added it. I'll remove it at the same time as your other suggestion.

Should the copyright be updated to 2021, or any other changes be made?

Reply:
> Should the copyright be updated to 2021, or any other changes be made?

I've no clue on this. I would say yes to the copyright but, in truth, doc comments seem largely aspirational and not enforced, so that's a question for @gregcorbett I think.

Reply:
In theory, you should add each year that a release is made, collapsing them into a range where appropriate. e.g. releases in 2001, 2002, 2003, 2005 gives 2001-2003, 2005

This is something that should probably be done at release time.

Reply:
We should at least get rid of the non-displayable character here.